demos/asp-net-core/ITOps.ViewModelComposition/Subscription-generic.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Http;
```

Comment:
we already had this conversation a few weeks ago 😄 at that time there was a `Subscription` class, that is now the `ISubscription` interface. `Subscription`1.cs` is not the convention, other than being horrible and hard to type on on en-US keyboards, is the way decompilers do that.

I can rename it to `Subscription.cs`